Update upgrade.rst - update grammar on upgrading to minor version before major version (#12062)
nextcloud/documentation: admin_manual/maintenance/upgrade.rst

--------------------

Nextcloud must be upgraded step by step: 

>   * Before you can upgrade to the next major release, you need to upgrade to the latest minor version for your current major version.

Reviewer: ```suggestion
  * Before you can upgrade to the next major release, you need to upgrade to the latest point release of your current major version.
```
Nextcloud usually doesn't have minor releases, only patch releases. This wording includes both if there ever is another minor release in the future.
